docs/animations/Expressions.md

```diff
@@ -0,0 +1,110 @@
+# ExpressionBuilder 
+Welcome to the ExpressionBuilder classes! The ExpressionBuilder classes are a C#-only alternative to building Expressions with type safety. Below is a quick introduction to using the ExpressionBuilder classes with your application. For complete documentation and walkthroughs, please see the Word Document in the project folder titled "ExpressionBuilder_Documentation.docx".
```

Comment:
word doc file can be found at https://github.com/Microsoft/WindowsUIDevLabs/blob/master/ExpressionBuilder/Docs/ExpressionBuilder_Documentation.docx
How should we bring this in?